tutorial/chapter/1/index.md

```diff
@@ -17,80 +14,52 @@ Squiggle can be installed via npm:
 
-Now any files ending in *.sqg* will be highlighted automatically as Squiggle
-code.
+If you use [Vim][vim], there's also a [Vim syntax highlighter plugin on GitHub][vimplug].
 
-## Built-in linter
-
-Many programming languages have tools built around them called linters. Usually
-these tools advise you of potential mistakes, or othwerwise ill-advised
-techniques. Frequently these tools require configuration before use, and are
-completely separate from the normal process of using the language.
+Now any files ending in `.sqg` will be highlighted automatically as Squiggle code.
 
-In Squiggle, good code is paramount, so linting is built-in to the compiler, and
-non-configurable. This ensures all projects using Squiggle will adhere to common
-guidelines.
-
-In addition, Squiggle is designed so that at the language level there are as few
-"gotchas" as possible, reducing the amount of linting necessary.
+## Built-in linter
 
-## Utility functions
+Many programming languages have tools built around them called linters. Usually these tools advise you of potential mistakes, or othwerwise ill-advised techniques. Frequently these tools require configuration before use, and are completely separate from the normal process of using the language.
 
-There are plans to eventually include a library of utility functions to use with
-Squiggle, in the spirit of [lodash][], but better suited for Squiggle.
+In Squiggle, good code is paramount, so linting is built-in to the compiler, and non-configurable. This ensures all projects using Squiggle will adhere to common guidelines.
 
-The current idea is a few simple functions that don't pass extra parameters that
-are usually ignored, that never mutate data, and always return frozen data.
+In addition, Squiggle is designed so that at the language level there are as few "gotchas" as possible, reducing the amount of linting necessary.
 
 ## Using with Node.js
 
-Currently the best way to use Squiggle with Node.js is to (unfortunately)
-compile the Squiggle in advance and store it in a folder and keep it in your Git
-project.
-
-I'm not generally a fan of committing generated files, but I don't think it's
-possible to generate them at installation time for your package.
-
-Here is an example compilation script in Bash:
-
-```bash
-#!/usr/bin/env bash
-set -e
-cd src
-for f in *.squiggle; do
-    name=$(basename "$f" .squiggle)
-    squiggle "$f" -o "../out/$name.js"
-done
-```
+You may be familiar with CoffeeScript or JSX hooks into Node.js's `require` function. This behavior is deprecated in Node.js and will not be included in Squiggle. The correct way to use Squiggle code with Node.js is to compile it ahead of time and put the resulting JavaScript code in another directory.
 
 Squiggle has a keyword for creating Node.js modules: `export`
 
 ```squiggle
-export 1
+let x = 1
+export x
 ```
 
-This is equivalent to the following JavaScript:
+This is like the following JavaScript:
```

Comment:
I'm trying to avoid saying that because it doesn't translate 100% the same… Maybe "works like"?